gateway/modules/schema/helpers.go

```diff
@@ -352,17 +365,43 @@ func (c *creationModule) modifyColumn() []string {
 	return queries
 }
 
-// modifyColumnType drop the column then creates a new column with provided type
-func (c *creationModule) modifyColumnType(dbType string) []string {
-	queries := []string{}
+func (c *creationModule) removeDirectives() []string {
+	var queries []string
 
 	if c.currentColumnInfo.IsForeign {
 		queries = append(queries, c.removeForeignKey()...)
+		c.currentColumnInfo.IsForeign = false
```

Comment:
specify a comment why this is important

Reply:
We just removed the foreign key. I think its pretty straight forward why i did that. Was to mark foreign key as removed to prevent this from being processed again